cmd/podman/containers/create.go

```diff
@@ -224,6 +224,10 @@ func CreateInit(c *cobra.Command, vals entities.ContainerCreateOptions, isInfra
 
 		if c.Flags().Changed("pids-limit") {
 			val := c.Flag("pids-limit").Value.String()
+			// Convert -1 to 0, so that -1 maps to unlimited pids limit
+			if val == "-1" {
+				val = "0"
```

Comment:
Do we know if Docker does something special on 0? Do they treat it literally, and the container can't fork at all?

Reply:
Tested with docker, and then set the pids limit to unlimited value when set to 0 as well - both -1 and 0 sets it to unlimited.

![Screenshot from 2021-09-30 15-05-41](https://user-images.githubusercontent.com/29377163/135515727-efba1e68-36bb-480c-90b8-30d1d9696263.png)